nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -284,6 +284,14 @@ let
         ${buildMenuAdditionalParamsGrub2 config "video=1920x1080@60"}
       }
 
+      # If we boot into a graphical environment where X is autorunned
```

Comment:
(it's not an issue per se) should it be autoran? or rephrased? English is weird, even weirderer when neologisms are introduced! ;)

Reply:
I believe it's autoran, but it appears I've used multiple alternates even in the commit messages.